styleguide/_themes/global/scss/swatch.scss

```diff
@@ -1,27 +1,33 @@
 //Path to CDN
 $images: 'https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com';
 
-//Colors
-$white: #fff;
-$light-gray-white: #efefef;
-$light-gray: #dedede;
-$medium-light-gray: #b8b8b8;
-$medium-gray: #666;
-$medium-dark-gray: #4c4c4c;
-$dark-gray: #333;
-$darkest-gray: #2a2a2a;
-$blue-gray: #2e3238;
-$black: #000;
+//primary colors
```

Comment:
@WidgetsBurritos  I'll switch out the medium-teal with teal-medium but the signup-green and rackspace-red is how brand refers to them so we're going to stick with that naming for consistency with brand.